docs/src/api/class-elementhandle.md

```diff
@@ -383,6 +387,9 @@ Optional event-specific initialization properties.
 
 ## async method: ElementHandle.evalOnSelector
 * since: v1.9
+* discouraged: This method does not wait for the element to pass actionability
```

Comment:
I think this is not the direct equivalent, because it requires a chained locator (this + selector).

Reply:
discussed offline, locators.md should explain it well enough!